src/utils/IdealGraphVisualizer/Coordinator/src/main/java/com/sun/hotspot/igv/coordinator/OutlineTopComponent.java

```diff
@@ -222,12 +225,22 @@ protected boolean requestFocusInWindow(boolean temporary) {
         return super.requestFocusInWindow(temporary);
     }
 
+    private GraphNode[] selectedGraphs = new GraphNode[0];
+    private final Set<FolderNode> selectedFolders = new HashSet<>();
```

Comment:
You should move these fields up to the other field declarations.

Reply:
done